pootle/apps/pootle_store/views.py

```diff
@@ -611,7 +618,30 @@ def submit(request, unit):
                     similarity=form.cleaned_data["similarity"],
                     mt_similarity=form.cleaned_data["mt_similarity"],
                 )
-                sub.save()
```

Comment:
Previously, we would create a `Submission()` object, then run a `save()` method on it that would store it in the database. Now we're creating this object but not doing anything with it, since the insertion happens inline in the code below.

I suggest moving the submission logic to the `Submission.save()` method, replacing the old code that should no longer be used, anyway, as I understand it. This will be cleaner, and will guarantee that submissions are always saved with the new method.

Reply:
I wasn't sure if we needed the new logic on every submission save or only on this view. I'll move it then, definitely cleaner